Content.Server/Medical/CryoPodWireAction.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Content.Server.Medical.Components;
+using Content.Server.Wires;
+using Content.Shared.Medical.Cryogenics;
+using Content.Shared.Wires;
+
+namespace Content.Server.Medical;
+
+/// <summary>
+/// Causes a failure in the cryo pod ejection system when cut. A crowbar will be needed to pry open the pod.
+/// </summary>
+[DataDefinition]
+public sealed class CryoPodWireAction: BaseWireAction
```

Comment:
could change the name of this to be something like CryoPodEjectLockWireAction, this is a little vague